### PR TITLE
Add support to check LTSS repos

### PR DESCRIPTION
**Pull request by mitiao**
verified run:
http://147.2.207.208/tests/491#step/repos_check/16

### CODE REVIEW

File: lib/utils.pm [hunk 1]
```diff
@@ -588,6 +588,9 @@ sub validatelr {
         $product = 'SLE-Module-Web-Scripting';
         $version = '12';
     }
+    if ($product =~ /LTSS/) {
+        $version = '';
```
Comment (okurz) [inline lib/utils.pm]: can you add a comment why this should be a good idea?

File: lib/utils.pm [hunk 2]
```diff
@@ -743,6 +752,10 @@ sub validate_repos_sle {
             for my $product_channel ("Pool", "Updates", "Debuginfo-Pool", "Debuginfo-Updates", "Source-Pool") {
                 # Toolchain module doesn't have Source-Pool channel
                 next if (($scc_product eq 'SLE-TCM') && ($product_channel eq 'Source-Pool'));
+                # LTSS doesn't have Pool, Debuginfo and Sourcer-Pool channels
```
Comment (okurz) [inline lib/utils.pm]: s/Sourcer/Source/

File: lib/utils.pm [hunk 3]
```diff
@@ -743,6 +752,10 @@ sub validate_repos_sle {
...
+                next if (($scc_product =~ /LTSS/) && ($product_channel eq 'Pool'));
+                next if (($scc_product =~ /LTSS/) && ($product_channel eq 'Debuginfo-Pool'));
+                next if (($scc_product =~ /LTSS/) && ($product_channel eq 'Source-Pool'));
```
Comment (okurz) [inline lib/utils.pm]: how about

```perl
next if $scc_product =~ /LTSS/ and $product_channel =~ /(|Debuginfo-|Source-)Pool/;
```

Comment (mitiao) [inline lib/utils.pm]: yeah, this better

File: lib/utils.pm (hunk 2 above)
Comment (mitiao) [inline lib/utils.pm]: fixed

File: lib/utils.pm (hunk 1 above)
Comment (mitiao) [inline lib/utils.pm]: added

**Pull request merged**